README.md

```diff
@@ -8,7 +8,13 @@ More information about the protocol in this [presentation](https://youtu.be/Jj8r
 ## Quick start - CLI
 
 1. Download the [latest `swap` binary release](https://github.com/comit-network/xmr-btc-swap/releases/latest) for your operating system
-2. Run the binary: `./swap buy-xmr --receive-address <YOUR MONERO ADDRESS>`
+2. Run the binary specifying the monero address where you wish to receive monero and the connection details of the seller:
+       ```
+       ./swap buy-xmr --receive-address <YOUR MONERO ADDRESS> --seller-peer-id <SELLERS PEER ID> --seller-multiaddr <SELLERS MULTIADDRESS>
+       ```
+   You can generate a receive address using your monero wallet. The seller will provide you their peer id and multiaddress.
+   We are running an `asb` instance on testnet you can swap with to get familiar with the `swap` CLI.
```

Comment:
Not quite a proper sentence I am afraid :sweat_smile: 

```suggestion
   We are running an `asb` instance on testnet.
   You can use it to get familiar with the `swap` CLI.
```

Reply:
I was intending for it to be single sentence but I think 2 sentences reads better as you have suggested.